tests/lib/User/DeletedUserTest.php

```diff
@@ -137,6 +137,10 @@ public function testCanChangeDisplayName() {
 		$this->assertFalse($this->deletedUser->canChangeDisplayName());
 	}
 
+	public function testCanMailAddressName() {
```

Comment:
```suggestion
	public function testCanChangeMailAddress() {
```

The function name seemed a bit scrambled.